test/graph/resource_wrapper.go

```diff
@@ -116,6 +118,16 @@ func addRoleByName(w baseWrapper, resource *resource.Resource, resourceTypeName
 	require.NoError(w.graph.t, err)
 }
 
+func removeRoleByName(w baseWrapper, resource *resource.Resource, resourceTypeName string, identityID uuid.UUID, roleName string) {
```

Comment:
This function (and `addRoleByName()` also for that matter) seem out of place here, considering they're not even used by the resource wrapper.

Reply:
Well, the `addRoleByName()` function is declared just after the `AddRole` on the `resourceWrapper`, and it can be used on `organization` and `spaces`, for example. So I thought it would make sense to apply the same logic, i.e., declaring the `removeRoleByName()` here, and then have the `RemoveAdmin` method on the `spaceWrapper` call this utility function.